opal/mca/mpool/base/mpool_base_alloc.c

```diff
@@ -74,7 +74,7 @@ void *mca_mpool_base_alloc(size_t size, opal_info_t *info, const char *hints)
 
     mpool = mca_mpool_base_module_lookup (hints);
     if (NULL != mpool) {
-        mem = mpool->mpool_alloc (mpool, size, 0, 0);
+        mem = mpool->mpool_alloc (mpool, size, sizeof(void *), 0);
```

Comment:
If `posix_memalign` requires a power of 2 for the alignment argument, what if `sizeof(void*)` == 4?

Reply:
4 is a power of two (2^2). But this will take the malloc code path anyway, since malloc is all you need for <= sizeof(void *) alignment.

Reply:
Um.. duh.  Right.  /me goes to get more coffee.